studio/schemas/documents/compensations.ts

```diff
@@ -29,6 +44,48 @@ const compensations = defineType({
       of: [{ type: benefitId }],
     }),
   ],
+  preview: {
+    select: {
+      title: "basicTitle",
+      /*
+       Array object values are accessed with dot notation and array index: "offices.0.basicTitle"
+       This limits the preview to only select a subset of the array
+       https://www.sanity.io/docs/previews-list-views#62febb15a63a
+      */
```

Comment:
I'd keep it more simple:

```suggestion
      // Dynamically selects office names for preview based on the OFFICES_PREVIEW_CUTOFF
```

Reply:
I think the original one is much more useful. The simpler one provides very little insight. It's hard to predict what a future developer will need, but to me the syntax was unexpected and called for some extra context.

Reply:
```suggestion
/*
   Access array object values using dot notation with the array index, e.g., "locations.0.basicTitle".
   This approach allows selecting a subset of the array for preview purposes.
   For more details, see: https://www.sanity.io/docs/previews-list-views#62febb15a63a
*/
```